### PR TITLE
Move performance support files out of eng/common

### PR DESCRIPTION
**Pull request by billwert**

### CODE REVIEW

File: eng/common/templates/job/performance.yml [hunk 1]
```diff
@@ -71,11 +71,11 @@ jobs:
         # Run all of the steps to setup repo
         - ${{ each step in parameters.steps }}:
           - ${{ step }}
-        - powershell: $(Build.SourcesDirectory)\eng\common\performance\performance-setup.ps1 $(IsInternal) -Framework $(_Framework) ${{ parameters.extraSetupParameters }}
+        - powershell: $(Build.SourcesDirectory)\eng\testing\performance\performance-setup.ps1 $(IsInternal) -Framework $(_Framework) ${{ parameters.extraSetupParameters }}
```
Comment (safern) [inline eng/common/templates/job/performance.yml:74]: These changes should be done in arcade as these yml templates are under common. 

It feels weird that common yml templates reference a folder outside of common.

**Comment by billwert** [comment]
Good point. I should have just also moved them. I will. Thanks for catching that.

Thanks,
-Bill
________________________________

File: eng/common/templates/job/performance.yml (hunk 1 above)
Comment (billwert) [reply eng/common/templates/job/performance.yml:74]: Turns out `performance.yml` is actually dead code. I moved the one we are still using (and will make an Arcade PR to delete them all.)

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 2]
```diff
@@ -111,18 +111,18 @@ jobs:
...
-    - script: $(Build.SourcesDirectory)/eng/common/performance/performance-setup.sh $(IsInternal)$(Interpreter) --framework $(_Framework) --kind ${{ parameters.runKind }} --logicalmachine ${{ parameters.logicalMachine }} ${{ parameters.extraSetupParameters }}
+    - script: $(Build.SourcesDirectory)/eng/testing/performance/performance-setup.sh $(IsInternal)$(Interpreter) --framework $(_Framework) --kind ${{ parameters.runKind }} --logicalmachine ${{ parameters.logicalMachine }} ${{ parameters.extraSetupParameters }}
       displayName: Performance Setup (Unix)
       condition: and(succeeded(), ne(variables['Agent.Os'], 'Windows_NT'))
       continueOnError: ${{ parameters.continueOnError }}
     - script: $(Python) $(PerformanceDirectory)/scripts/ci_setup.py $(SetupArguments)
       displayName: Run ci setup script
       # Run perf testing in helix
-    - template: /eng/common/templates/steps/perf-send-to-helix.yml
+    - template: /eng/pipelines/coreclr/templates/perf-send-to-helix.yml
```
Comment (DrewScoggins) [inline eng/pipelines/coreclr/templates/run-performance-job.yml:125]: You will need to move the yml file from common to its new location.

**Review by DrewScoggins — CHANGES_REQUESTED** [review]
Need to update the location of the `perf-send-to-helix.yml`

**Review by DrewScoggins — APPROVED** [review]

File: eng/pipelines/coreclr/templates/run-performance-job.yml (hunk 2 above)
Comment (DrewScoggins) [reply eng/pipelines/coreclr/templates/run-performance-job.yml:125]: Missed this because of relocate and change

**Pull request merged**